generator/src/main/java/org/corfudb/generator/operations/NestedTxOperation.java

```diff
@@ -0,0 +1,66 @@
+package org.corfudb.generator.operations;
+
+import ch.qos.logback.classic.Logger;
+import lombok.extern.slf4j.Slf4j;
+import org.corfudb.generator.Correctness;
+import org.corfudb.generator.State;
+import org.corfudb.runtime.exceptions.TransactionAbortedException;
+import org.slf4j.LoggerFactory;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.slf4j.LoggerFactory'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)